src/ess.jl

```diff
@@ -328,10 +336,12 @@ function ess_rhat(
         sum_pₜ = pₜ
 
         k = 2
-        while k < maxlag
+        while true
```

Comment:
Any reason why we do not just use
```suggestion
        for k in 2:2:(maxlag - 1)
```
here?

Reply:
Yes, if we move the final `ρ_even` computation after the loop (suggested below and now implemented), it's useful to declare `k` before the loop.